extension-requests/style.css

```diff
@@ -503,9 +503,19 @@ body {
   align-items: center;
   justify-content: center;
   width: 9rem;
-  height: 100%;
+  height: 2.5rem;
   padding: 0.7rem;
 }
+.sort-button {
+  background-color: var(--light-gray-color);
+  border-radius: 0.4rem;
+  border: 2.5px solid var(--black-color);
+  padding: 8px;
+  height: 2.5rem;
```

Comment:
- padding could have been used in `rem` units whereas height should generally be a fixed number.

Reply:
We are following a convention where any value larger than 8px are written in rem. @sahsisunny can provide more context on this decision.

Reply:
- what is the reasoning behind this @sahsisunny?

Reply:
We can keep some values in px if we want to fix size. for others like the value is less than 10px we can keep them in PX to avoid decimal.

Reply:
Its, not a convention.